target/linux/ath79/dts/qca9563_zte_mf282.dts

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2021 Cezary Jackiewicz
+// Copyright (c) 2021, 2022 Lech Perczak
+// Copyright (c) 2022 David Bauer <mail@david-bauer.net>
+// Copyright (c) 2023 Andreas Böhler <dev@aboehler.at>
+
+#include "qca9563_zte_mf28x.dtsi"
+
+/ {
+	model = "ZTE MF282";
+	compatible = "zte,mf282", "qca,qca9563";
+
+	ubi-concat {
+		compatible = "mtd-concat";
+		devices = <&ubiconcat0 &ubiconcat1>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				reg = <0x0 0x7440000>;
+				label = "ubi";
+			};
+		};
+	};
+};
+
+&led_debug {
```

Comment:
This needs adding to MF281 device tree, with blue changed to green.

Reply:
Thanks, fixed.